### DISCUSSION TITLE
Trying to only display specific Directives but its not limiting the Builtin directives.

### DISCUSSION THREAD

**Discussion by gglissendorf** [thatmattlove/hyperglass#309] [Q&A] [answered]
I want to only display traceroute and ping on some devices.  I have tried doing by specifying the built-in directive only.  I also created my own directive and tried to limit all the built-in directives.  Both methods I still see all the Built-in Directives.  Am I missing something?

  - name: Ping Source Zayo MSP
    address: 10.13.0.232
    platform: nokia_sros
    directives:
        - builtin: false
        - tier1-ping  **(with this method I see the tier1-ping and all the builtin still)**
    credential:
  - name: Ping Source NTT DEN
    address: 10.13.0.233
    platform: nokia_sros
    directives:
        - builtin: [NokiaSROS_Ping, hyperglass_nokia_sros_ping]
    credential:

**Comment by richardtector** [comment, marked as answer]
Try 'builtins:' instead of 'builtin:' - there's a typo in the docs, and a PR to fix it here: https://github.com/thatmattlove/hyperglass/pull/281

**Reply by gglissendorf** [reply]
worked thanks.
